build/Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM golang:1.23.3-alpine3.20 AS builder
+FROM golang:1.23.12-alpine3.21 AS builder
 
 WORKDIR /go/src/av-scan-service
 COPY go.mod go.sum main.go ./
 COPY pkg ./pkg/
 RUN go build -o av-scan-service . 
 
-FROM alpine:3.20
+FROM alpine:3.21
+
+RUN apk update && apk upgrade && apk add --no-cache clamav
```

Comment:
ClamAV is not part of this image. ClamAV is a separate image inside the helm chart